cloud/services/compute/image.go

```diff
@@ -36,15 +39,37 @@ func (s *Service) GetImage(imageId string) (*osc.Image, error) {
 	}
 	oscApiClient := s.scope.GetApi()
 	oscAuthClient := s.scope.GetAuth()
-	readImageResponse, httpRes, err := oscApiClient.ImageApi.ReadImages(oscAuthClient).ReadImagesRequest(readImageRequest).Execute()
-	if err != nil {
-		fmt.Printf("Error with http result %s", httpRes.Status)
-		return nil, err
+	var readImagesResponse osc.ReadImagesResponse
+	readImageCallBack := func() (bool, error) {
+		var httpRes *_nethttp.Response
+		var err error
+		readImagesResponse, httpRes, err = oscApiClient.ImageApi.ReadImages(oscAuthClient).ReadImagesRequest(readImageRequest).Execute()
+		if err != nil {
+			if httpRes != nil {
+				fmt.Printf("Error with http result %s", httpRes.Status)
+			}
+			requestStr := fmt.Sprintf("%v", readImageRequest)
+			allowedErrors := map[string]int{"AccessDenied": 503}
```

Comment:
```"AccessDenied": 503``` should be refactorize

Reply:
Add the "429" for throttling (in case of changes)

Reply:
Ok